CodeStarterCamp_Week4/main.swift

```diff
@@ -8,5 +8,12 @@
 
 import Foundation
 
-print("Hello, World!")
+var person = BodyCondition(upperBodyStrength: 0, lowerBodyStrength: 0, muscularEndurance: 0, fatigue: 0)
+
+let exercise: Array<Exercise> = [sitUp, sitUp, sitUp, activeRest, deadLift, deadLift, deadLift, activeRest, plank, plank, plank, activeRest, barbellRow, barbellRow, barbellRow, activeRest, cablePullover, cablePullover, cablePullover, activeRest, longRun]
+
+var hellRoutine = Routine(routineName: "지옥의 헬스루틴", exercises: exercise)
```

Comment:
- 여기선 person 인스턴스는 반드시 `var`여야 하죠~?
구조체로 만들었으니까요. ㅎㅎㅎ
대신에 새로 만든 짧은 이니셜라이저를 한 번 적용해보세요! 👍🏻 

- exercise 를 선언할 때, 타입 명시를 `[Exercise]` 로 바꿔봤습니다.
근데 여전히 Array<Exercise> 가 좋다면, 그걸 사용하셔도 괜찮아요. 😄 

- hellRoutine 인스턴스는 var → `let` 으로 선언해도 전혀 문제가 없네요!
항상 let 을 기본으로 생각하시되, 에러가 뜨면 그때 var 로 바꾸는 걸 추천드립니다!
`변경`될 필요가 없는 객체들을 var 로 선언해두면, 다른 개발자가 볼 때 오해할 수도 있으니까요.
확실하게 `"얘는 변경되지 않는 애다!"` 느낌을 주기 위해 let 으로 선언해두면, 코드 읽기가 더 편해요. 🌊

```suggestion
var person = BodyCondition(startingPoint: 0) // 새로 만든 짧은 이니셜라이저 적용해봄

let exercise: [Exercise] = [sitUp, sitUp, sitUp, activeRest, deadLift, deadLift, deadLift, activeRest, plank, plank, plank, activeRest, barbellRow, barbellRow, barbellRow, activeRest, cablePullover, cablePullover, cablePullover, activeRest, longRun]

let hellRoutine = Routine(routineName: "지옥의 헬스루틴", exercises: exercise) // let 으로 바꾸기
```